crosswalks-sssom/zenodo/zenodo-codemeta-mappings.yml

```diff
@@ -0,0 +1,14 @@
+---
+curie_map: # This contains the URLs prefixes used in the mapping file
+  skos: http://www.w3.org/2004/02/skos/core# ### Always needed. Do not modify
+  sssom: https://w3id.org/sssom/             ### Always needed. Do not modify
+  subject: "https://github.com/zenodo/zenodo/blob/master/zenodo/modules/deposit/jsonschemas/deposits/records/legacyrecord.json"
+  subject_documentation: "https://developers.zenodo.org/"
```

Comment:
We have put this outside the curie_map, just in case

Reply:
Also, this does not point to a version no?

Reply:
No, we currently don't version the REST API/metadata documentation. But once we have the next metadata version live we'll make sure we have different URL paths for them.

Reply:
I see, this will have to do then